src/utils.js

```diff
@@ -0,0 +1,17 @@
+/* @flow */
+import type { LocaleType } from '@paypal/sdk-constants/src/';
+import { COUNTRY_LANGS, LANG, COUNTRY } from '@paypal/sdk-constants/src';
+
+export function getComputedLocales(locale : string) : LocaleType {
+    const [ _lang, _country ] = locale.split('_');
```

Comment:
I think we can avoid the extra variables by using `let` here. Ex:
```js
let [ lang, country ] = locale.split('_');

lang = LANG[lang.toUpperCase()];
country = COUNTRY[country];
const countryLangs = COUNTRY_LANGS[country];
```

Reply:
my bad, fixed, thanks!